doc/getting_started/setup_fpga.md

```diff
@@ -257,16 +257,67 @@ To load `hello_world` into the FPGA on the ChipWhisperer CW310 board follow the
 
 #### Troubleshooting
 
-If the firmware load fails, try pressing the "USB RST" button before loading the bitstream.
+If the firmware load fails, try pressing the "USR-RST" button before loading the bitstream.
 
 ## Connect with OpenOCD and debug
 
-To connect the ChipWhisperer CW310 FPGA board with OpenOCD, run the following command
+The CW310 supports JTAG-based debugging with OpenOCD and GDB via the standard ARM JTAG headers on the board (labeled USR Debug Headers).
+To use it, program the bitstream and bootstrap the desired firmware, then connect a JTAG adapter to one of the headers.
+For this guide, the `Olimex ARM-USB-TINY-H` JTAG adapter was used.
+
+After bootstrapping the firmware, the TAP straps may need to be set.
+As of this writing, the FPGA images are typically programmed to be in the RMA lifecycle state, and the TAP straps are sampled continuously in that state.
+To connect the JTAG chain to the CPU's TAP, adjust the strap values with opentitantool.
+Assuming opentitantool has been built and that the current directory is the root of the workspace, run these commands:
+
+```console
+$ ./bazel-bin/sw/host/opentitantool/opentitantool \
+        --conf sw/host/opentitantool/config/opentitan_cw310.json \
+        --interface cw310 \
+        gpio write TAP_STRAP0 false
+$ ./bazel-bin/sw/host/opentitantool \
```

Comment:
./bazel-bin/sw/host/opentitantool/**opentitantool**

Reply:
Oops, missed that edit when I moved away from `bazel run`. Thanks!